react-js-buy/src/components/Product.js

```diff
@@ -10,13 +10,9 @@ class Product extends Component {
       defaultOptionValues[selector.name] = selector.values[0].value;
     });
     this.state = { selectedOptions: defaultOptionValues };
-
-    this.handleOptionChange = this.handleOptionChange.bind(this);
-    this.handleQuantityChange = this.handleQuantityChange.bind(this);
-    this.findImage = this.findImage.bind(this);
   }
 
-  findImage(images, variantId) {
+  findImage = (images, variantId) => {
```

Comment:
Please don't change code unnecessarily. Function declarations are used for named functions for the sake of consistency.